robotidy/decorators.py

```diff
@@ -67,3 +70,24 @@ def decorator(cls):
         return cls
 
     return decorator
+
+
+def return_node_untouched(node):
+    return node
+
+
+def check_start_end_line(func):
+    """
+    Do not transform node if it's not withing passed start_line and end_line.
```

Comment:
```suggestion
    Do not transform node if it's not within passed start_line and end_line.
```

Reply:
I based this PR on another PR that's now open so I'll fix there and then update this branch after merging. 
I still need some extra testing as this solution is failing on Linux but it's working fine on Windows